app.js

```diff
@@ -78,9 +78,23 @@ function camSizeChange(width, height) {
 var loadParams = getLoadParams();
 var modelParams = getModelParams();
 
+function setCameraDevices(devices) {
```

Comment:
I'm very confused the usage of 4 functions:
1. setCameraDevices()
2. setCameraOptions()
3. setCameraSelection()
4. setCamera()

you may merge [1,2] and [3,4]

Reply:
Yeah makes sense, done.